providers/sakuracloud/sakuracloudProvider.go

```diff
@@ -0,0 +1,105 @@
+package sakuracloud
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/StackExchange/dnscontrol/v4/models"
+	"github.com/StackExchange/dnscontrol/v4/providers"
+)
+
+const defaultEndpoint = "https://secure.sakura.ad.jp/cloud/zone/is1a/api/cloud/1.1"
+
+var features = providers.DocumentationNotes{
+	// The default for unlisted capabilities is 'Cannot'.
+	// See providers/capabilities.go for the entire list of capabilities.
+	providers.CanAutoDNSSEC:          providers.Cannot(),
+	providers.CanConcur:              providers.Cannot(),
+	providers.CanGetZones:            providers.Can(),
+	providers.CanUseAlias:            providers.Can(),
+	providers.CanUseCAA:              providers.Can(),
+	providers.CanUseDHCID:            providers.Cannot(),
+	providers.CanUseDNAME:            providers.Cannot(),
+	providers.CanUseDS:               providers.Cannot(),
+	providers.CanUseDSForChildren:    providers.Cannot(),
+	providers.CanUseHTTPS:            providers.Can(),
+	providers.CanUseLOC:              providers.Cannot(),
+	providers.CanUseNAPTR:            providers.Cannot(),
+	providers.CanUsePTR:              providers.Can(),
+	providers.CanUseSOA:              providers.Cannot(),
+	providers.CanUseSRV:              providers.Can(),
+	providers.CanUseSSHFP:            providers.Cannot(),
+	providers.CanUseSVCB:             providers.Can(),
+	providers.CanUseTLSA:             providers.Cannot(),
+	providers.CanUseDNSKEY:           providers.Cannot(),
+	providers.DocCreateDomains:       providers.Can(),
+	providers.DocDualHost:            providers.Cannot(),
+	providers.DocOfficiallySupported: providers.Cannot(),
+}
+
+func init() {
+	fns := providers.DspFuncs{
+		Initializer:   newSakuracloudDsp,
+		RecordAuditor: AuditRecords,
+	}
+	providers.RegisterDomainServiceProviderType("SAKURACLOUD", fns, features)
```

Comment:
**Change**: The function `RegisterMaintainer()` needs to be included so that the `OWNERS` is generated successfully.

```go
        const providerMaintainer = "@ttkzw"
	providers.RegisterMaintainer(providerName, providerMaintainer)
```

See also the other examples: https://github.com/search?q=repo%3AStackExchange%2Fdnscontrol+path%3A%2F%5Eproviders%5C%2F%2F+const+providerMaintainer&type=code

Reply:
Added `RegisterMaintainer()`.

[SAKURACLOUD: Add RegisterMaintainer()](https://github.com/StackExchange/dnscontrol/pull/3086/commits/ac861cf9df61b5dd3873c8e89e17466bc8bcb6d9)

Reply:
Awesome! 🤩